src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
+
+
+function App() {
+ 
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading]=React.useState(true);
+  useEffect(() => {
+    new Promise((resolve, reject)=>{
+      setTimeout(() => {
+        resolve({data:{
+          todoList:JSON.parse(localStorage.getItem("savedTodoList")),
+        }})
+      }, 2000);
+
+    }) 
+    .then((result)=>{
+     setTodoList(result.data.todoList);
+     setIsLoading(false);
+    })
+  },[])
+
+  useEffect(() => {
+    if(!isLoading){
+    localStorage.setItem("savedTodoList", JSON.stringify(todoList));}
+  }, [todoList]
```

Comment:
`isLoading` should also be a dependency of the `useEffect` because you're referencing it.  So, it should be `[todoList, isLoading]`